language-php.md

```diff
@@ -44,6 +44,21 @@ Yes:
 $variableName
 ```
 
+### Function (Only PHP7)
+The function we write should declare parameter type and return type, also apply to function annotation.
```

Comment:
How do you mean "apply to function annotation"?

Reply:
I don't think we have a rule saying PHPDoc is mandatory. We can add that too, but I think it's a separate issue, and not only applies to PHP7.